lzy/graph-executor-2/src/main/java/ai/lzy/graph/model/TaskSlotDescription.java

```diff
@@ -0,0 +1,44 @@
+package ai.lzy.graph.model;
+
+import java.util.List;
+import java.util.Map;
+
+public record TaskSlotDescription(
+    String name,
+    String description,
+    String poolLabel,
+    String zone,
+    String command,
+    List<Slot> slots,
+    Map<String, String> slotsToChannelsAssignments,
+    String stdoutName,
```

Comment:
stdout/stderr slots were removed about a month ago :)
now we always pass stdout/stderr via kafka